content/plugins/loader-options-plugin.md

```diff
@@ -0,0 +1,37 @@
+---
+title: loader-options-plugin
+---
+
+?> Review this content
+
+The `loader-options-plugin` is unlike other plugins.  It exists to help people move from webpack 1 to webpack 2.  With webpack 2 the schema for a `webpack.config.js` became stricter; no longer open for extension by other loaders / plugins.  With webpack 2 the intention is that you pass `options` directly to loaders / plugins. i.e. options are **not** global / shared.
+
+However, until a loader has been updated to depend upon options being passed directly to them, the `loader-options-plugin` exists to bridge the gap.  You can configure global / shared loader options with this plugin and all loaders will receive these options.
+
+In the future this plugin may be removed.
+
+```javascript
+new webpack.LoaderOptionsPlugin(options)
+```
+
+* `options.debug` (`boolean`): Whether loaders should be in `debug` mode or not.
+* `options.minimize` (`boolean`): Where loaders can be switched to minimize mode.
+* `options.options` (`object`): A configuration object that can be used to configure older loaders - this will take the same schema a `webpack.config.js`
```

Comment:
*sass-loader*, *less-loader* at least. You would have to look up their readme files to see if they have root level configuration in place as this plugin literally replaces that.